packages/date-helpers/__tests__/date-helpers.test.ts

```diff
@@ -1,9 +1,65 @@
 import { parseISO } from 'date-fns';
 import { DateHelpers } from '../src/date-helpers';
+import { InvalidArgumentException } from '@rightcapital/exceptions';
 
 const output = parseISO('2019-10-21');
 
-describe('DateHelpers', () => {
+describe('DateHelpers input validation', () => {
+  it('parseDateString should throw for unrecognized date formats', () => {
+    const invalidDateString = 'ABCDE';
+    expect(() => DateHelpers.parseDateString(invalidDateString)).toThrowError(
+      InvalidArgumentException,
+    );
+  });
+
+  it('parseDateString should throw error for null or undefined input', () => {
+    // @ts-expect-error - intentionally passing invalid input to test error handling
+    expect(() => DateHelpers.parseDateString(null)).toThrow(
```

Comment:
```suggestion
    expect(() => DateHelpers.parseDateString(null as unknown as DateLike)).toThrow(
```

Reply:
`as unknown as string`? I believe @ts-expect-error is clearer for intentional test errors.🤔

Reply:
In this particular scenario, we are indeed passing a `null` value, which does not conform to the expected `DateLike` type. Therefore, it appears that using `@ts-expect-error` would be more appropriate. 🤔